internal/output/html/style.css

```diff
@@ -119,19 +124,22 @@ header {
   background-color: rgba(255, 255, 255, 0.04);
 }
 
+.table-tr-details > td {
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+}
+
+.vuln-tr {
+  display: table-row;
+}
+
 .vuln-id {
   color: #d9534f;
 }
 
 .package-details {
   text-align: left;
-  padding: 0 50px;
-  display: none;
-  padding-top: 20px;
-  padding-bottom: 30px;
-  border-bottom: 1px solid rgba(255, 255, 255, 0.12);
-  max-height: 800px;
```

Comment:
I still think we should keep the max height and make the package details a scrollable window. This would be a better user experience for packages with many vulnerabilities. I see the tooltip issue, but I feel it's not too much problem, the aliases/group ID tooltip window can still be viewed by scrolling down.